desc/integrals/singularities.py

```diff
@@ -134,21 +140,56 @@ def _heuristic_support_params(grid):
 
     """
     assert grid.can_fft2
-    # TODO: Account for real space grid anisotropy by choosing
-    #   ratio = |e_zeta|/|e_theta|
-    #   ratio_avg = surface_average(ratio, jacobian=|e_theta x e_zeta|)
-    #   st/(sz*NFP) (at evaluation point i) = mean (ratio(i), ratio_avg).
-    #   Then we have ~circle in real space around each singular point.
-    #   The grid resolution can then still be chosen independently according
-    #   to the frequency content of R, Z, λ to net the best function approximation.
+    # This is simple and should improve convergence, but do later.
```

Comment:
if there is an issue for this, add to the comment, and if not, make an issue for using the local ratio